content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -469,6 +469,10 @@ with the following command:
 $ dvc exp gc --workspace --force
 ```
 
+Checkout the video covering this Checkpoints doc.
+
+https://youtu.be/PcDo-hCvYpw
+
 Now that you know how to use checkpoints in DVC, you'll be able to resume
```

Comment:
Should it be near the top instead? Not sure people who've read the full thing will find the video interesting at that point.

Reply:
Probably no need for the intro sentence too. I'd just drop the video in a logical place.